apps/api/config.ts

```diff
@@ -1,12 +1,62 @@
-import { existsSync } from 'node:fs';
+import { existsSync, readFileSync } from 'node:fs';
```

Comment:
don't use sync, yes I know it's on startup. still.

Reply:
But it's on startup.. why not?

Reply:
Because nodejs, async etc... sync api should really never be used.